lingua_franca/internal.py

```diff
@@ -18,24 +18,79 @@
                                  "fa-ir", "pt-pt", "ru-ru", "sl-si",
                                  "sv-se", "tr-tr", "eu-eu")
 
```

Comment:
please use `bidict` here as sometimes we need a key by value

Reply:
By definition, we should always get the key as `val.split('-', 1)[0]` in this list